pdns/dnsdist.cc

```diff
@@ -1670,7 +1828,24 @@ static void MultipleMessagesUDPClientThread(ClientState* cs, LocalHolders& holde
 }
 #endif /* defined(HAVE_RECVMMSG) && defined(HAVE_SENDMMSG) && defined(MSG_WAITFORONE) */
 #endif /* DISABLE_RECVMMSG */
-
+static void xskClientThread(ClientState* cs)
```

Comment:
Yes, but not exactly right. My current idea is to have `xskClientThread` replace `udpClientThread` and `tcpClientThread` when `XDP` is enabled. The work on the `tcpClientThrea` part here has not started yet. This is just a current idea, and may change in the future.

Reply:
Understood! I wonder if that makes sense to support XDP for TCP in our case, do you know if the performance improvements are significant?

Reply:
No worry, especially since I now understand that handling UDP but not TCP would likely be hard, so it looks like we will have to handle XDP for TCP anyway.